docs/platform-services/automation-service/app-central/integrations/virustotal.md

```diff
@@ -7,25 +7,26 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('/img/platform-services/automation-service/app-central/logos/virustotal.png')} alt="virustotal" width="100"/>
 
-***Version: 1.12  
-Updated: June 21, 2024***
+***Version: 1.13  
+Updated: July 03, 2024***
 
 Perform threat intelligence evidence gathering with VirusTotal.
 
 ## Actions
 
 * **Add Comment** (*Notification*) - Add a new comment.
-* **Domain Reputation** (*Enrichment*) - Gather domain reputation information on a specific domain v2.
+* **Check Scanned Status** (*Scheduled*) - Check the status of a Scan File and Scan URL.
+* **Domain Reputation** (*Enrichment*) - Gather domain reputation information on a specific domain.
+* **Download Report** (*Enrichment*) - Download a file of scanned report.
 * **File Reputation** (*Enrichment*) - Gather reputation information on a specific file.
 * **Get Comment** (*Enrichment*) - Gather all comments.
+* **Get Report** (*Enrichment*) - Gather a report on a specific file or URL.
 * **IP Reputation** (*Enrichment*) - Gather IP reputation information for a specific IP address.
-* **IP Reputation V3** (*Enrichment*) - Gather IP reputation information for multiple IP addresses v3.
-* **Scan File** (*Enrichment*) - Gather information on a specific file.
-* **Scan URL** (*Enrichment*) - Gather information on a specific URL.
+* **Scan File** (*Enrichment*) - Scan the specific file.
+* **Scan URL** (*Enrichment*) - Scan on a specific URL.
 * **URL Reputation** (*Enrichment*) - Gather reputation information for multiple URLs.
-* **File Scan** **V2** (*Enrichment*) - Acquire information about a specific file without saving the report.
-* **URL Scan V2** (*Enrichment*) - Obtain information about a particular URL without storing the report.
-* **Search VirusTotal** (*Enrichment*) - Search for File Hash, URL, Domain, IP address and comments by tags.
+  <br/><img src={useBaseUrl('/img/platform-services/automation-service/app-central/integrations/virus-total/virus-total-1.png')} style={{border:'1px solid gray'}} alt="virus-total" width="800"/>
+  <br/><img src={useBaseUrl('/img/platform-services/automation-service/app-central/integrations/virus-total/virus-total-2.png')} style={{border:'1px solid gray'}} alt="virus-total" width="800"/>
```

Comment:
@rishav-sumo-dev, these two images are the same.

Reply:
Nope they different one is `Scan Url` & `Scan File`
<img width="788" alt="Screenshot 2024-07-03 at 9 29 54 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/161286338/484e42ce-5815-4acd-8908-00e1e18a6abe">

Reply:
Oh, I see! Thank you.